build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python src/etc/build_rust.py $@
```

Comment:
To keep the experience consistent across platforms I'd be fine just removing this small shell script (or moving a python script to the top of the project)